_includes/footer/footer.html

```diff
@@ -7,6 +7,9 @@
 
 <footer>
   {% assign content = site.data.footer %}
+  {% if include.careers %}
+    {% assign content = site.data.footer-careers %}
```

Comment:
Looks a little bit strange to me
Not sure about final idea, but at least - `subtitle`, `href`, `creator`, `icon` are not changing.
Only `title` and `email` change.
Maybe then pass only them through params?
or separate title-email section to some component (include)
This is just an idea, not final decision